.sync.yml

```diff
@@ -1,8 +1,15 @@
 ---
+.travis.yml:
+  docker_sets:
+    - set: ubuntu1404-64m
+    - set: ubuntu1604-64m
+    - set: centos6-64m
+    - set: centos7-64m
+    - set: debian8-64m
```

Comment:
why is there an `m` at the end of each line?

Reply:
Role `master` was needed for one of the beaker helpers. `run_task` IIRC.